plugin/storage/cassandra/cassandra3v001.cql

```diff
@@ -0,0 +1,155 @@
+CREATE KEYSPACE IF NOT EXISTS jaeger_v1_test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
```

Comment:
I would rather we port the shell script, which can work for real prod deployment. Just remove the Uber defaults from it.